cli/e2e.js

```diff
@@ -91,6 +85,7 @@ function spawnProtractor(chunks, port, skyPagesConfig) {
     protractorPath,
     [
       getProtractorConfigPath(),
+      `--disableChecks`,
```

Comment:
FYI, adding this check to solve the "unknown arguments" warning (soon to be error) that's shown.  https://github.com/angular/protractor/blob/master/CHANGELOG.md#features-3